libs/s25main/Settings.cpp

```diff
@@ -72,9 +72,10 @@ void Settings::LoadDefaults()
     // {
     // 0 = ask user at start,1 = enabled, 2 = disabled
     global.submit_debug_data = 0;
-    global.use_upnp = 2;
+    global.use_upnp = false;
```

Comment:
this breaks existing, did you increase the settings version?

Reply:
Well we assign this to a bool (even prior to this PR) so `2` is "true" which is likely not intended. Nothing is broken, the currently stored "2" will be converted to "true" on load and to "1" on save so this is still compatible, isn't it?

Reply:
ah true though, thats alright then.